### PR TITLE
Fixing ci failures

### PR DESCRIPTION
**Pull request by maqi**

### CODE REVIEW

File: resources/scripts/all_nodes_joined.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ echo "Checking logfiles to check all nodes have joined"
 log_dir="$HOME/.safe/node/local-test-network"
 
 # -u needed here to search log dirs
-nodes_joined=$(rg "ReceivedJoinApproval" "$log_dir" -g "*.log*"  -u -c | wc -l)
```
Comment (joshuef) [inline resources/scripts/all_nodes_joined.sh:20]: @maqi this line is using a `LogMarker`, so would be good to keep that in place. If we want to target another one, that's fine and we can add one in the right place for this

**Review by joshuef — CHANGES_REQUESTED** [review]

File: resources/scripts/all_nodes_joined.sh (hunk 1 above)
Comment (maqi) [reply resources/scripts/all_nodes_joined.sh:20]: 'ReceivedJoinApproval' is on Trace level, however the E2E full test starting nodes with log level of Info, which doesn't have it logged at all.
Hence I choosed another one that is logged as Info already.
I can change the place of 'ReceivedJoinApproval' to be on Info level, if you think that's better?

Comment (joshuef) [reply resources/scripts/all_nodes_joined.sh:20]: Yeh, I think so. 

But we should also have the nodes logging trace level I think. I don't think that's been intentionally disabled 🤔

Comment (maqi) [reply resources/scripts/all_nodes_joined.sh:20]: I thought it's for reduce the logs size uploaded to the github?
There are 33 nodes, log with Info could save over 98% space & upload/download time

**Review by joshuef — APPROVED** [review]

**Review by joshuef — APPROVED** [review]

**Pull request merged**